pkgs/top-level/python-packages.nix

```diff
@@ -12995,6 +12995,10 @@ in {
 
   pyopencl = callPackage ../development/python-modules/pyopencl { };
 
+  pyomo = callPackage ../development/python-modules/pyomo { };
+
+  pyutilib = callPackage ../development/python-modules/pyutilib { };
+
   pyproj = callPackage ../development/python-modules/pyproj {
```

Comment:
`pyproj` definitely belongs above `pyutilib`. The attributes should be ordered alphabetically.

Reply:
Right... but well `python-packages.nix` is a mess. 
Even if `pyutillib` goes after `pyproj`, there are for example `repoze_who` and `vobject` before `pyproj`, and `kaa-base` and `mmpython` after `pyproj`.
That make it quite tricky to find the right position.

Reply:
Yes, but we can still try